.github/workflows/linters.yml

```diff
@@ -28,10 +28,10 @@ jobs:
           node-version: "12.x"
       - name: Setup Webhint
         run: |
-          npm install --save-dev hint@6.x
+          npm install --save-dev hint@5.x
```

Comment:
**Changes Required**
- [ ] There should be no custom changes to this file. Please use the exact `linter configuration` provided. If you feel there should be an update, please raise a PR to the Microverse term.